packages/@react-spectrum/label/test/StaticField.test.js

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {CalendarDate, CalendarDateTime, Time, ZonedDateTime} from '@internationalized/date';
+import React from 'react';
+import {render} from '@react-spectrum/test-utils';
+import {StaticField} from '../src';
+
+describe('StaticField', function () {
+  it('renders correctly with string value', function () {
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        value="test" />
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(staticField).toBeInTheDocument();
+    expect(staticField).not.toHaveAttribute('aria-describedby');
```

Comment:
was this aria-describedby check a remnant from an older test? Just curious where it came from

Reply:
true, we do not really need this